### ISSUE TITLE
Fix message formatting on create_survey_boundaries_dhs()

### ISSUE THREAD

**Issue by jeffeaton** [mrc-ide/naomi.utils#7]
Format of this error message does not accommodate multiple surveys:

```r
> surveys <- create_surveys_dhs("RWA")
> survey_region_boundaries <- create_survey_boundaries_dhs(surveys)

SurveyId = RW2005DHSRW2015DHS contains multiple boundary levels.
>
```

**Comment by jeffeaton** [comment]
While at it, suppress these `sf` warnings and messages:

```r
> surveys <- surveys_add_dhs_regvar(surveys, survey_region_boundaries)
> survey_region_areas <- allocate_areas_survey_regions(areas_wide, survey_region_boundaries)
although coordinates are longitude/latitude, st_intersection assumes that they are planar
although coordinates are longitude/latitude, st_intersection assumes that they are planar
although coordinates are longitude/latitude, st_intersection assumes that they are planar
although coordinates are longitude/latitude, st_intersection assumes that they are planar
although coordinates are longitude/latitude, st_intersection assumes that they are planar
although coordinates are longitude/latitude, st_intersection assumes that they are planar
Warning messages:
1: attribute variables are assumed to be spatially constant throughout all geometries 
2: attribute variables are assumed to be spatially constant throughout all geometries 
3: attribute variables are assumed to be spatially constant throughout all geometries 
4: attribute variables are assumed to be spatially constant throughout all geometries 
5: attribute variables are assumed to be spatially constant throughout all geometries 
6: attribute variables are assumed to be spatially constant throughout all geometries 
```

**Comment by jeffeaton** [comment]
The second of these is addressed in 799b3fb.

**Comment by jeffeaton** [comment]
Resolved by 3434251.
